### PR TITLE
Fix a parsing issue of config in panels ##panels

### PR DESCRIPTION
**Pull request by Vane11ope**
way easier to deal with json for the panels config if the "caching" parameters were int and there's no problem doing it so.

### CODE REVIEW

File: libr/core/panels.c [hunk 1]
```diff
@@ -2946,7 +2946,7 @@ static void savePanelsLayout(RCore* core, bool temp) {
 		pj_kn (pj, "y", panel->pos.y);
 		pj_kn (pj, "w", panel->pos.w);
 		pj_kn (pj, "h", panel->pos.h);
-		pj_kb (pj, "caching", panel->caching);
+		pj_kn (pj, "caching", panel->caching);
```
Comment (radare) [inline libr/core/panels.c]: n expects ut64, this is int, and i dont see why you want caching to be int instead of bool

Comment (Vane11ope) [reply libr/core/panels.c]: so how do you pass char* "true" as bool true, i mean i can do it somehow but i can save it if i make it int at the first place

**Comment by radare** [comment]
wat? pj_kb should use true,false from !0 and 0.

**Comment by Vane11ope** [comment]
what do you even mean.

json is gonna look like {"Title":"Disassembly","Cmd":"pd $r","x":0,"y":1,"w":97,"h":78,"caching":false} then when i parse it, i get char* "false", how do i translate that to bool false, actual working code would help me understand a lot thanks.

**Comment by radare** [comment]
Pls dont make json camel case

**Comment by Vane11ope** [comment]
?????????

anyway a bug remains unless this gets merged or you show me alternative. 

i dont really understand why you stick to bool so much but i leave this here to you.

**Comment by radare** [comment]
How do u parse this to get a string out of a bool? Also if u just want to show a number 0,1 bool is your friend. So just changing 1 line will be enough. Keep all the other lines intact

**Pull request closed without merging**